manager/assets/modext/widgets/core/modx.grid.js

```diff
@@ -364,7 +364,10 @@ Ext.extend(MODx.grid.Grid,Ext.grid.EditorGridPanel,{
                 ,autoDestroy: true
                 ,listeners:{
                     load: function(){
-                        Ext.getCmp('modx-content').doLayout(); /* Fix layout bug with absolute positioning */
+                        var cmp = Ext.getCmp('modx-content');
+                        if(typeof cmp !== "undefined") {
+                            cmp.doLayout(); /* Fix layout bug with absolute positioning */
+                        }
```

Comment:
What you've done certainly works, but please consider this:

I'd like to encourage everyone to employ ES6+ syntaxes and methods when submitting new or updated js code. Generally the `var` keyword should be avoided whenever possible, opting for the newer `let` or `const` keywords. Also, since you're just testing for truthiness/falseyness of your `cmp` variable, you can simply do this:
```
const cmp = Ext.getCmp('modx-content');
if (cmp) {
    cmp.doLayout();
}
```

Reply:
Thanks for the suggestion, @smg6511! I've updated the PR accordingly (and verified manually on my local installation that the behavior is unchanged).